### ISSUE TITLE
A11y: Toast notifications not announced by assistive technologies

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/clients#2566]
### Steps To Reproduce

1. Execute an operation that results in a toast/notification (e.g. adding an item, failing to enter correct login credentials, etc)
2. See a toast notification appear in the top right corner

### Expected Result

When running a screen reader, the toast notification's content should be announced.

### Actual Result

Toast notifications are silent/not conveyed at all to assistive technologies

### Screenshots or Videos

![Added item toast](https://user-images.githubusercontent.com/895831/159796880-a9b73835-7ec4-49e6-ac95-308a06c96332.png)

![An error has occurred toast](https://user-images.githubusercontent.com/895831/159796924-695c5fe7-ce7b-4a1f-82fc-7836d7e83bc3.png)

### Additional Context

These toasts/notifications should use an `aria-live="assertive" aria-atomic="true"` region. Note that in order for live regions to make their announcement correctly, the region/container itself needs to be created first and be empty (so that the browser/AT know to start watching the container for changes), and *then* the content needs to be injected into it (at which point AT will then announce the change). See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

### Operating System

Windows

### Operating System Version

10

### Installation method

Direct Download (from bitwarden.com)

### Build Version

Version 1.32.1 Shell 16.0.7 Renderer 96.0.4664.110 Node 16.9.1 Architecture x64

**Comment by patrickhlauke** [comment]
related https://github.com/bitwarden/clients/issues/2509

**Comment by patrickhlauke** [comment]
and, as with the extension issue, I suspect the fix will come automagically once updated to latest version of ngx-toastr where I submitted an upstream fix.

**Comment by djsmith85** [comment]
@patrickhlauke The bump to Angular 14 and with that the bump of ngx-toastr to 15.0 was done with https://github.com/bitwarden/clients/pull/2948.

This will be available with the `2022.09` release.

**Comment by patrickhlauke** [comment]
ah, perfect, thanks @djsmith85 - once i get a moment, i'll build the latest main branch locally to verify 👍

**Comment by patrickhlauke** [comment]
confirming that notifications are now announced correctly by screen readers


https://user-images.githubusercontent.com/895831/183533428-25b9a70f-aa33-435a-8a66-86c5cf0a03c9.mp4

arguably, there's a bit of tweaking that could still be done, as they all announced with "Close" at the start (due to the "X" close control). but i'll file this as a separate issue/PR another time.
